quickstart-templates/application-serviceprincipal-create-client-resource/main.bicep

```diff
@@ -1,4 +1,4 @@
-provider microsoftGraph
+extension microsoftGraph
```

Comment:
For each of the Readmes, do we need to specify the minimum Bicep version number?

Reply:
Yeah we have required Bicep version number in the release notes, but I think it's a good idea to include in each quickstart Readme. I will update it